src/redux/reducers/btn.ts

```diff
@@ -0,0 +1,26 @@
+import { createSlice, type PayloadAction } from "@reduxjs/toolkit";
+
+interface BtnState {
+  value: string;
+}
+
+const initialStat2e: BtnState = {
+  value: "All", // JSON.parse(localStorage["reduxx"]).btn.value,
+};
+
+export const btnSlice = createSlice({
+  name: "btn",
+  initialState: initialStat2e,
+  reducers: {
+    setBtn: (state, action: { payload: { newBtn: string } }) => {
```

Comment:
Всегда передай сам payload в виде уже нужного тебе значения. В функции где вызываешь этот action напиши просто setBtn('blablabla') (строка произвольная
```suggestion
    setBtn: (state, { payload }}) => {
```